[CI] Static Code Analysis failed

## Description

### Current State

```bash
-- Configuring done
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
LLD_COMMON
    linked by target "wasmedgeAOT" in directory /home/runner/work/WasmEdge/WasmEdge/lib/aot
LLD_CORE
    linked by target "wasmedgeAOT" in directory /home/runner/work/WasmEdge/WasmEdge/lib/aot
LLD_DRIVER
    linked by target "wasmedgeAOT" in directory /home/runner/work/WasmEdge/WasmEdge/lib/aot
LLD_READERWRITER
    linked by target "wasmedgeAOT" in directory /home/runner/work/WasmEdge/WasmEdge/lib/aot
LLD_SYSTEM
    linked by target "wasmedgeAOT" in directory /home/runner/work/WasmEdge/WasmEdge/lib/aot
LLD_YAML
    linked by target "wasmedgeAOT" in directory /home/runner/work/WasmEdge/WasmEdge/lib/aot

-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
External Error: *** capture command failed:
*** cmake -DCMAKE_BUILD_TYPE=Debug ..
*** exited with code 1
```

### Expected

The CI of Static Code Analysis should pass

--- a/.github/workflows/static-code-analysis.yml
+++ b/.github/workflows/static-code-analysis.yml
@@ -34,7 +34,7 @@ jobs:
     
     - name: Install Dependencies
       run: |
-        sudo apt install -y software-properties-common libboost-all-dev llvm-12-dev liblld-12-dev clang-12 tar
+        sudo apt install -y software-properties-common libboost-all-dev llvm-11-dev liblld-11-dev clang-11 tar
         curl -sSL "https://github.com/facebook/infer/releases/download/v1.1.0/infer-linux64-v1.1.0.tar.xz" | sudo tar -C /opt -xJ && sudo ln -s "/opt/infer-linux64-v1.1.0/bin/infer" /usr/local/bin/infer
         
 

